### ISSUE TITLE
i18n support

### ISSUE THREAD

**Issue by luochen1990** [bigcapitalhq/bigcapital#183]
Do we support multiple languages in the User Interface?

**Comment by abouolia** [comment]
Yeah it supports localization, currently just localized it for English and Arabic but we welcome to any external contributors to contribute to localize the application to any other languages.

**Comment by quangngoc** [comment]
I'd love to have french language in bigcapital

**Comment by wolone** [comment]
How does the newly added language take effect

**Comment by elbeicktalat** [comment]
Well You do support Arabic, but during the setup and even after I see only English why?

I can help with localizing big capital in the Italian language.

**Comment by abouolia** [comment]
@elbeicktalat Because some Arabic keywords still not translated, but soon Arabic will be in the list, if you want to contribute and translate it to Italian you can do it through [Transifex](https://transifex.com/bigcapital/bigcapital)https://transifex.com/bigcapital/bigcapital/ if you needed any help or question please let me directly. thanks for your contribution in advance.
